### ISSUE TITLE
Error loading from Solution

### ISSUE THREAD

**Issue by JBoman32768** [MscrmTools/MscrmTools.Iconator#8]
I have my favourite solution I have been working on  - but when I select "Load from Solution" I get a Object not set to a reference of an object error :(

"Load All" works okay - but makes me nervous as I want the image associations to become part a particular Solution.

![image](https://user-images.githubusercontent.com/8730120/79943452-0fa40100-84a8-11ea-9b05-15e391a81ce3.png)

**Comment by MscrmTools** [comment]
Never faced this issue but I will try to reproduce... can you tell me what is your environment version?

In the meantime, loading all just mean the items (entities and images) displayed are not coming only from the solutions you select.

If you then use only items from your solution(s), there is nothing to fear

**Comment by MscrmTools** [comment]
Can you test with this file? 

[MsCrmTools.Iconator.zip](https://github.com/MscrmTools/MscrmTools.Iconator/files/4517671/MsCrmTools.Iconator.zip)

**Comment by MscrmTools** [comment]
@JBoman32768 ping, to be sure you don't miss my previous comment

**Comment by JBoman32768** [comment]
Sorry for the delay – will take a look.

**Comment by JBoman32768** [comment]
So I copied the DLL overwriting AppData\Roaming\MscrmTools\XrmToolBox\Plugins\MsCrmTools.Iconator.dll

Slightly different error message: Error while loading CRM components.

[A screenshot of a social media post  Description automatically generated]

However when I changed it back to the original version it has started working.  I have switched it backward and forward and the error only happens with the file you sent me now – seems to be fixed in the nuget version.

Thanks 😊

J.

From: James Boman
Sent: Monday, 27 April 2020 9:59 AM
To: MscrmTools/MscrmTools.Iconator <reply@reply.github.com>
Subject: RE: [MscrmTools/MscrmTools.Iconator] Error loading from Solution (#8)

Sorry for the delay – will take a look.
